azure-nextgen-cs-credential-rotation-one-set/AppStack.cs

```diff
@@ -0,0 +1,304 @@
+// Copyright 2016-2020, Pulumi Corporation.  All rights reserved.
+
+using System;
+using Pulumi;
+using Pulumi.AzureNextGen.Authorization.Latest;
+using Pulumi.AzureNextGen.EventGrid.V20200401Preview.Inputs;
+using Pulumi.AzureNextGen.EventGrid.V20200401Preview;
+using Pulumi.AzureNextGen.Insights.Latest;
+using Pulumi.AzureNextGen.KeyVault.Latest;
+using Pulumi.AzureNextGen.KeyVault.Latest.Inputs;
+using Pulumi.AzureNextGen.Resources.Latest;
+using Pulumi.AzureNextGen.Sql.Latest;
+using Pulumi.AzureNextGen.Storage.Latest;
+using Pulumi.AzureNextGen.Web.Latest;
+using Pulumi.AzureNextGen.Web.Latest.Inputs;
+using KeyVault = Pulumi.AzureNextGen.KeyVault.Latest;
+using Storage = Pulumi.AzureNextGen.Storage.Latest;
+using ManagedServiceIdentityType = Pulumi.AzureNextGen.Web.Latest.ManagedServiceIdentityType;
+
+internal class AppStack : Stack
+{
+    [Output]
+    public Output<string> WebAppEndpoint { get; set; }
+
+    public AppStack()
+    {
+        var config = new Config();
+        var resourceGroupName = config.Get("resourceGroupName") ?? "rotatesecretoneset";
+        var resourceNamePrefix = config.Get("resourceNamePrefix") ?? resourceGroupName;
+        var location = config.Get("location") ?? "EastUS";
+
+        var resourceGroup = new ResourceGroup("resourceGroup", new ResourceGroupArgs { ResourceGroupName = resourceGroupName, Location = location });
+
+        var sqlAdminLogin = config.Get("sqlAdminLogin") ?? "sqlAdmin";
+        var sqlAdminPassword = Guid.NewGuid().ToString();
```

Comment:
This will change on every run, so I expect you see a diff on every `pulumi up`. It's better to use the random provider for this (or just hard-code for simplicity).

Reply:
Yeah, I did that because I thought it would be more secure if no one ever knew what the password was. But the password changes within a minute anyway, so maybe it's over the top. :)

Reply:
Fixed